### ISSUE TITLE
Unable to use Macros on Windows

### ISSUE THREAD

**Issue by 1337MARCEL** [oven-sh/bun#10128]
### What version of Bun is running?

1.1.3+2615dc742

### What platform is your computer?

Microsoft Windows NT 10.0.22631.0 x64

### What steps can reproduce the bug?

index.ts

```ts
import { echo } from "./utils" with { type: "macro" };

console.log(await echo());
```

utils.ts

```ts
import { $ } from "bun";

export async function echo() {
  return await $`echo "Hello, World!"`.text();
}
```

### What is the expected behavior?

```
$ bun run index.ts
Hello, World!
```

### What do you see instead?

```
$ bun run index.ts
Segmentation fault at address 0x8
???:?:?: 0x7ff6a7ada22b in ??? (bun.exe)
???:?:?: 0x7ff6a7f4e736 in ??? (bun.exe)
???:?:?: 0x7ff6a7f4f07b in ??? (bun.exe)
???:?:?: 0x7ff6a7bd9f2a in ??? (bun.exe)
???:?:?: 0x7ff6a84b8682 in ??? (bun.exe)
???:?:?: 0x7ff6a83f06ea in ??? (bun.exe)
???:?:?: 0x7ff6a8206980 in ??? (bun.exe)
???:?:?: 0x7ff6a8100e54 in ??? (bun.exe)
???:?:?: 0x7ff6a8206adc in ??? (bun.exe)
???:?:?: 0x7ff6a8100e54 in ??? (bun.exe)
???:?:?: 0x7ff6a8206adc in ??? (bun.exe)
???:?:?: 0x7ff6a8206764 in ??? (bun.exe)
???:?:?: 0x7ff6a7f4f2e4 in ??? (bun.exe)
???:?:?: 0x7ff6a7ada0eb in ??? (bun.exe)
???:?:?: 0x7ff6a7bcdb91 in ??? (bun.exe)
???:?:?: 0x7ff6a9f11e8a in ??? (bun.exe)
???:?:?: 0x205a042119d in ??? (???)
???:?:?: 0x43cbb62f7f in ??? (???)
???:?:?: 0x2059d8b33ff in ??? (???)
???:?:?: 0x43cbb632ff in ??? (???)
???:?:?: 0x205e0a208cf in ??? (???)
???:?:?: 0x43cbb62f7f in ??? (???)
???:?:?: 0x7ff6aa211104 in ??? (bun.exe)
```

**Comment by redraskal** [comment]
I'm only unable to use async macros on Windows

**Comment by 1337MARCEL** [comment]
## Output from the new crash reporter

Bun v1.1.5 ([#10527](https://github.com/oven-sh/bun/pull/10527)) on windows x86_64 [RunCommand]

Segmentation fault at address 0x00000068

- [`event_loop.zig:2168`](https://github.com/oven-sh/bun/blob/b257a309772db2e9dfb745f97068aea6a32ca151/src/bun.js/event_loop.zig#L2168): `loop`
- [`interpreter.zig:910`](https://github.com/oven-sh/bun/blob/b257a309772db2e9dfb745f97068aea6a32ca151/src/shell/interpreter.zig#L910): `writeFailingErrorFmt`
- [`writer.zig:17`](https://github.com/oven-sh/bun/blob/b257a309772db2e9dfb745f97068aea6a32ca151/src/deps/zig/lib/std/io/writer.zig#L17): `write`
- [`glob.zig:1480`](https://github.com/oven-sh/bun/blob/b257a309772db2e9dfb745f97068aea6a32ca151/src/glob.zig#L1480): `buildPatternComponents`
- [`writer.zig:17`](https://github.com/oven-sh/bun/blob/b257a309772db2e9dfb745f97068aea6a32ca151/src/deps/zig/lib/std/io/writer.zig#L17): `write`
- [`glob.zig:1480`](https://github.com/oven-sh/bun/blob/b257a309772db2e9dfb745f97068aea6a32ca151/src/glob.zig#L1480): `buildPatternComponents`
- [`writer.zig:17`](https://github.com/oven-sh/bun/blob/b257a309772db2e9dfb745f97068aea6a32ca151/src/deps/zig/lib/std/io/writer.zig#L17): `write`
- [`glob.zig:1480`](https://github.com/oven-sh/bun/blob/b257a309772db2e9dfb745f97068aea6a32ca151/src/glob.zig#L1480): `buildPatternComponents`
- [`writer.zig:17`](https://github.com/oven-sh/bun/blob/b257a309772db2e9dfb745f97068aea6a32ca151/src/deps/zig/lib/std/io/writer.zig#L17): `write`
- [`buffered_writer.zig:23`](https://github.com/oven-sh/bun/blob/b257a309772db2e9dfb745f97068aea6a32ca151/src/deps/zig/lib/std/io/buffered_writer.zig#L23): `writer`

**Comment by dylan-conway** [comment]
This was fixed in bun v1.1.21
